packages/typescript-resolver-files/src/generateResolverFiles/generateResolverFiles.ts

```diff
@@ -119,6 +119,21 @@ export const generateResolverFiles = (
     );
   });
 
+  // Enum
+  Object.values(
+    ctx.config.parsedGraphQLSchemaMeta.userDefinedSchemaTypeMap.enum
+  ).forEach((details) => {
+    visitNamedType(
+      {
+        ...details,
+        belongsToRootObject: null,
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
+        visitor: () => {}, // FIXME: Enum are not being generated and should be if needed: (1) has mapper OR (2) in the right filesystem location. This is here to make sure externalResolver works for enums.
+      },
+      ctx
+    );
+  });
```

Comment:
Before v0.9, we send all schema types to `vistNamedType` which checks for external resolvers before trying to call the `visitor` function.

From v0.9, we do all the parsing of schema types and put them in `userDefinedSchemaTypeMap`. This way is better as we only need to parse the schema once, the order / intention of handling certain types are more explicit. However, we need to manually add the rest of types here. 

Looks like the only missing type right now is `Enum`, so this block should now exhaustively handle all possible types.